sig-architecture/charter.md

```diff
@@ -1,40 +1,175 @@
-
-* Defining the scope of the Kubernetes project
-  * [What is (and is not) Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)
-* Maintaining, evolving, and enforcing the deprecation policy
-  * [Deprecation policy](https://kubernetes.io/docs/reference/deprecation-policy/)
-* Documenting and evolving the system architecture
-  * [Kubernetes Design and Architecture](../contributors/design-proposals/architecture/architecture.md)
-* Defining and driving necessary extensibility points
-* Establishing and documenting design principles
-  * [Design principles](../contributors/design-proposals/architecture/principles.md)
-* Establishing and documenting conventions for system and user-facing APIs
-  * [API conventions](/contributors/devel/api-conventions.md)
-* Developing necessary technical review processes, such as the proposal and API review processes
-* Driving improvement of overall code organization, including github orgs and repositories
-* Educating approvers/owners of other SIGs (e.g., by holding office hours)
-
-Out of scope:
-* Issues specific to a particular component or functional area, which would be the purview
-  of some other SIG, except where they deviate from project-wide principles and conventions.
-* [Release support policy](/contributors/design-proposals/release/versioning.md)
-  is owned by SIG Release
-
-TODO:
-* Formalize decision processes
-* Document initial reviewers and approvers
-* Clarify criteria for areas out of scope for the SIG
-* Document who owns client library, build, and release artifacts
-* Document who owns conformance definition, profiles, etc.
+## Mission
+
+The Architecture SIG maintains and evolves the design principles of Kubernetes, and provides a consistent body of expertise necessary to ensure architectural consistency over time.
+
+## Scope and subprojects
+
+The scope of SIG Architecture covers issues that span all the system's components, how the code is organized, how the components fit together, how they interact, what common conventions they follow, general requirements and restrictions, overall technical scope of the project, etc. SIG Architecture is also responsible for educating members of other SIGs about these policies and principles. 
+
+SIG Architecture is also a good place to identify gaps in the project, collect thinking on how the project should approach those problems and then work to motivate that those problems get addressed. This will often take the form of working to draw attention through an appropriate SIG.
+
+Issues specific to a particular component or functional area, which would be the purview of some other single SIG, are out of scope for SIG Architecture, except where they deviate from project-wide principles and conventions, or are not yet being addressed, as discussed above.
+
+SIG Architecture’s subprojects are as follows. [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml) contains links to their OWNERS.
+
+<table>
```

Comment:
You can do table's in markdown:

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

Let's avoid HTML if at all possible.

Reply:
Can we just expand the sigs.yaml schema to generate these tables? 


Sigs.yaml is already generating https://github.com/kubernetes/community/tree/master/sig-architecture#subprojects